### DISCUSSION TITLE
Ability to set markers

### DISCUSSION THREAD

**Discussion by jamessewell** [marcusolsson/grafana-gantt-panel#84] [Ideas]
Still loving this plugin!

Would it be possible to have a type which rather than being a bar was marked by an icon of some sort?

I have a lot of events (at a time, not a duration) which I'd like to plot.

**Comment by marcusolsson** [comment]
Interesting! So like annotations basically?

**Comment by jamessewell** [comment]
Yeah! I think milestones is the term Gantt chart people use.

https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQH2E7wS78iHtUMLwiPRqm0eXA4cUGJ5fdtAfwxJCG7uAcYN0M_
